app/lib/cms/s3.rb

```diff
@@ -35,7 +35,9 @@ def region
     end
 
     def credentials
-      config.slice(:access_key_id, :secret_access_key) if enabled?
+      return unless enabled?
```

Comment:
Maybe this behavior already has a test but good to add such if not.

Reply:
Look at that... It wasn't tested at all. Added in fd980729e.